src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1029,46 +1029,46 @@ else if (fieldClass == Pattern[].class) {
             result = "Regular Expressions";
         }
         else if (fieldClass == SeverityLevel.class) {
-            result = "Severity";
+            result = "SeverityLevel";
```

Comment:
It seems to me all this could be reduced to:
````
} else if (fieldClass.isEnum()) {
  result = fieldClass.getSimpleName();
}
````

Reply:
Done